src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/delete_interaction_choice_set_request.cc

```diff
@@ -159,7 +211,9 @@ void DeleteInteractionChoiceSetRequest::SendVrDeleteCommand(
   choice_set = &((*choice_set)[strings::choice_set]);
   for (uint32_t i = 0; i < (*choice_set).length(); ++i) {
     msg_params[strings::cmd_id] = (*choice_set)[i][strings::choice_id];
-    SendHMIRequest(hmi_apis::FunctionID::VR_DeleteCommand, &msg_params);
+    const std::uint32_t sent_request = SendHMIRequest(
```

Comment:
@IGapchuk rename to something like `delete_command_corr_id`

Reply:
@AKalinich-Luxoft Renamed in e32b004ae4d7e84e153d6b8082cb3c1a800a41ec.